### PR TITLE
Review project

### PR DESCRIPTION
**Pull request by gfpoliva**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -10,9 +10,12 @@ jobs:
       - uses: actions/checkout@v2
       - name: Cypress evaluator step
         id: cypress
-        uses: betrybe/cypress-evaluator-action@v1
+        uses: betrybe/cypress-evaluator-action@v2
+        with:
+          repository-test-name: 'betrybe/sd-03-block5-project-color-guess'
```
Comment (LucasCFerraz) [inline .github/workflows/main.yml:15]: here should be sd-0x...tests, ok?

File: README.md [hunk 2]
```diff
@@ -4,177 +4,219 @@ Você já usa o GitHub diariamente para desenvolver os exercícios, certo? Agora
...
+Neste projeto, você implementará um adivinhador de cor RGB. Ou seja, dada uma string RGB, você irá fornecer 6 alternativas visuais de cores, sendo uma delas a correta! Você poderá estilizar seu projeto da **forma que desejar** desde que todos os requisitos sejam cumpridos. Use sua imaginação! 🧠
 
-### 6 - Deve haver um botão para reiniciar o jogo
+---
 
-Ao clicar no botão, deve ser sorteada uma nova cor RGB e novas cores para as bolas, além de voltar os textos para a forma inicial.
+## Requisitos do projeto
 
-Este botão também poderá ser utilizado para iniciar um novo jogo.
+### 1 - O seu site deve ser completamente centralizado
```
Comment (LucasCFerraz) [inline README.md:132]: They dont saw flex by now. What do you think?

Comment (LucasCFerraz) [reply README.md:132]: If you want to keep this requirement, I think we should put it as bonus.

File: .github/workflows/main.yml (hunk 1 above)
Comment (gfpoliva) [reply .github/workflows/main.yml]: Fixed in 6298e4c o/

File: README.md (hunk 2 above)
Comment (gfpoliva) [reply README.md]: Set as a bonus in a34f19e :D

**Pull request merged**